Bottom: ed7a5a27c08bceaff18c6df55fa715e8dc8ca6f5
Top:    3ce8262a437aa3904b90b0b5c31238f60882ecbe
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-27 10:46:17 -0500

Add Changepassword.php page to phpldapadmin



---
diff --git a/docker/docker-phpldapadmin/Dockerfile b/docker/docker-phpldapadmin/Dockerfile
index 4544e647d..249051d65 100644
--- a/docker/docker-phpldapadmin/Dockerfile
+++ b/docker/docker-phpldapadmin/Dockerfile
@@ -1,3 +1,3 @@
 FROM osixia/phpldapadmin:latest
 
-COPY changepassword.php /var/www/changepassword.php
+COPY changepassword.php /var/www/phpldapadmin_bootstrap/htdocs/changepassword.php
